addons/stock/models/stock_orderpoint.py

```diff
@@ -180,11 +180,20 @@ def _onchange_route_id(self):
         if self.route_id:
             self.qty_multiple = self._get_qty_multiple_to_order()
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        if any(val.get('snoozed_until', False) and val.get('trigger', 'auto') == 'auto' for val in vals_list):
```

Comment:
```suggestion
        if any(val.get('snoozed_until', False) and val.get('trigger', self.default_get('trigger')) == 'auto' for val in vals_list):
```
So it would crash if someone modify the configuration later

Reply:
or you could put everything in a helper function and call it after write and create to avoid code duplication

Reply:
I always forget that I am not payed by the line ^^

Reply:
@amoyaux I added the default getter but I didn't find any cleaver way to use a nice helper method (the condition in the write and create seems quite different to me).